PROPOSED-fsmon-config.plist

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Version</key>
+	<string>v0.2-PROPOSED</string>
+	<key>WatchItems</key>
+	<dict>
+		<key>my_watcher</key>
+		<dict>
+			<key>Paths</key>
+      <array>
+        <!-- Paths can be strings with defaults set (IsPrefix=false) -->
+        <string>/Users/*/path1</string>
+        <!-- Paths can also be dicts with keys to override defautls -->
+        <!-- Currently one attribute, IsPrefix, maybe more in the future -->
+        <dict>
+          <key>Path</key>
+          <string>/Users/*/path2</string>
+          <key>IsPrefix</key>
+          <true/>
+        </dict>
+      </array>
+      <!-- Options that apply to the whole rule are now grouped together to -->
+      <!-- make more clear they apply to the whole rule, not a path. -->
+      <key>Options</key>
+      <dict>
+        <!-- Key changed from `WriteOnly`, makes more clear how this affects -->
+        <!-- runtime (e.g. OPEN vs CLONEFILE "read access") -->
+        <key>AllowReadAccess</key>
+        <false/>
+        <key>AuditOnly</key>
+        <false/>
+        <!-- Expandable to support future options. Potential upcoming ones: -->
+        <!-- Potential upcoming options: -->
+        <!--   * InvertProcessLogic: By default, listed processes are allowed -->
+        <!--     to perform the op. this key could invert that logic such that -->
+        <!--     the listed processes are denied and everything else is allowed. -->
+        <!--   * ProcessLogicAND: By default, a process matching any of the criteria -->
+        <!--     is allowed. This key =would require a process meet all the criteria. -->
+      </dict>
+      <key>Processes</key>
```

Comment:
Another potential name here would be `Exceptions` instead of `Processes`. This would provide more flexibility for attributes we want to add in the future. A potential downside is interoperability with other tools that only operate on process exceptions and would make a common superset config more difficult (can discuss offline).

Reply:
When we're describing exceptions are we `and`-ing or `or`-ing the key value pairs?

Reply:
The current default is `or`. Above I commented on a potential future `ProcessLogicAND` key in the `Options` dictionary.